plugins/kotlin/plugin/k1/resources/META-INF/kotlin-core-fe10.xml

```diff
@@ -193,6 +193,7 @@
     <copyPastePostProcessor implementation="org.jetbrains.kotlin.idea.conversion.copy.ConvertTextJavaCopyPasteProcessor"/>
     <copyPastePostProcessor implementation="org.jetbrains.kotlin.idea.codeInsight.KotlinCopyPasteReferenceProcessor"/>
     <copyPastePreProcessor implementation="org.jetbrains.kotlin.idea.editor.KotlinLiteralCopyPasteProcessor"/>
+    <copyPastePreProcessor implementation="org.jetbrains.kotlin.idea.editor.KDocCopyPastePreProcessor"/>
```

Comment:
The code doesn't depend on resolve, so it should be located in `plugins/kotlin/plugin/common/resources/META-INF/kotlin-core.xml`

Reply:
Fixed.